CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Added
 
+- [PR #333](https://github.com/konpyutaika/nifikop/pull/333) - **[Operator]** Replace Update by Patch on K8S resource.
```

Comment:
```suggestion
- [PR #333](https://github.com/konpyutaika/nifikop/pull/333) - **[Operator]** Replace Update by Patch on K8S resource to avoid update race conditions.
```